CONTRIBUTING.md

```diff
@@ -58,23 +58,36 @@ ensure consistent code style and quality.
 ## Tests
 
 Tests and lint checks are automatically run on pull requests and pushes to the main
-branch
-using GitHub Actions.
+branch using GitHub Actions.
 
 To run the tests locally, run the following command from the root directory of the
-repository
-while the virtual environment is active:
+repository while the virtual environment is active:
 
 ```bash
 pytest
 ```
 
+To run lint checks locally:
+
+```bash
+ruff check .
+```
+
+To automatically fix linting issues:
+
+```bash
+ruff check --fix .
+```
+
+To automatically format the code:
+
+```bash
+ruff format .
+```
+
```

Comment:
Would it make sense to show how to pre-commit locally (a la `pre-commit run -a`) since we have the  configuration?

Reply:
Done!